src/assets/scss/gallery.scss

```diff
@@ -0,0 +1,70 @@
+.gallery {
+  @include backgroundImg("/assets/img/bg-gallery.png");
+  display: flex;
+  justify-content: space-between;
+
+  &__list {
+    padding: 137px 0 98px 99px;
+
+    &-title {
+      padding-bottom: 37px;
+      font-family: Yeseva One, serif;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 27px;
+      line-height: 37px;
+      text-transform: uppercase;
+      color: #E3B873;
```

Comment:
There is a variable `$yellow: #E3B873;` in var file. So please use the variables if you defined (there are also same cases with white color and font-family)